### DISCUSSION TITLE
What is the general workflow in building an ontology?

### DISCUSSION THREAD

**Discussion by JisooSeo** [JisooSeo/101-WG-Discussions#6] [Q&A] [answered]
What is the general workflow in building an ontology?

**Comment by JisooSeo** [comment]
I'm not saying that this is the only way to build an ontology, but Bill Duncan taught me that you first need to start a spreadsheet that has a list of terms, IRIs, and definitions. And then you add content and do the backend techie stuff. I will add to this thread as I research more!

**Reply by giacomodecolle** [reply, marked as answer]
That's one way to do it. I have worked in a similar way with Alex Diehl. Other times I also directly edit the .owl or .ttl file, but it's true that if you have a big list of terms having them written down in an excel sheet first is very helpful. You can also set up a template to directly ingest terms from the excel sheet to the ontology through ROBOT, which is an OBO tool. It's also true that sometimes people just want to add to your ontology, and having too many barriers to go through (e.g. edit the excel file instead of the ontology directly) might be prevent them from doing so. Remember that ontology is a collaborative project. If we are talking about broader general workflow, there is much more to say. I'd say the first steps are to understand what problems the ontology can solve, what questions it should be able to answer once data is in it, and then create your ontology architecture around it (plus starting from BFO ofc). To do some self-promotion, see also https://philarchive.org/rec/DECODS-3

**Reply by JisooSeo** [reply]
Your paper is such a great resource! :D Thank you for the comment! I completely agree with you that the very first step involves the discussion of the purpose of the ontology.
